app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -13,7 +13,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Example activity to verify identity in Bluetooth connections.
+ * Used to let the user verify his identity by tapping on certain places in an image
```

Comment:
Class JavaDoc should make sense without the class itself, so could you start with "{@link Activity} that..."